static/app/gettingStartedDocs/python/starlette.tsx

```diff
@@ -70,67 +83,81 @@ const onboarding: OnboardingConfig = {
       ],
     },
   ],
-  configure: (params: Params) => [
-    {
-      type: StepType.CONFIGURE,
-      description: tct(
-        'If you have the [codeStarlette:starlette] package in your dependencies, the Starlette integration will be enabled automatically when you initialize the Sentry SDK. Initialize the Sentry SDK before your app has been initialized:',
-        {
-          codeStarlette: <code />,
-        }
-      ),
-      configurations: [
-        {
-          language: 'python',
-          code: `${getSdkSetupSnippet(params)}
+  configure: (params: Params) => {
+    const profilingMode = params.organization.features.includes('continuous-profiling')
+      ? 'continuous'
+      : 'transaction';
+
+    return [
+      {
+        type: StepType.CONFIGURE,
+        description: tct(
+          'If you have the [codeStarlette:starlette] package in your dependencies, the Starlette integration will be enabled automatically when you initialize the Sentry SDK. Initialize the Sentry SDK before your app has been initialized:',
+          {
+            codeStarlette: <code />,
+          }
+        ),
+        configurations: [
+          {
+            language: 'python',
+            code: `
+${getSdkSetupSnippet(params, profilingMode)}
 app = Starlette(routes=[...])
-        `,
-        },
-      ],
-    },
-  ],
-  verify: (params: Params) => [
-    {
-      type: StepType.VERIFY,
-      description: t(
-        'You can easily verify your Sentry installation by creating a route that triggers an error:'
-      ),
-      configurations: [
-        {
-          language: 'python',
+`,
+          },
+        ],
+        additionalInfo: <AlternativeConfiguration />,
+      },
+    ];
+  },
+  verify: (params: Params) => {
+    const profilingMode = params.organization.features.includes('continuous-profiling')
+      ? 'continuous'
+      : 'transaction';
```

Comment:
nit: Why not pass the profiling mode as part of the params?
It would avoid the often repeated feature check logic / type definitions and would preserve the current function interfaces.